help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Acxiom Audience Distribution
+description: Use the [!DNL Acxiom Audience Distribution] destination to enhance audiences with [!DNL Acxiom's Real ID] technology and activate audiences to multiple platforms, such as [!DNL Altice], [!DNL Ampersand], [!DNL Comcast], and more.
+badge: Beta
+---
+# [!DNL Acxiom Audience Distribution] destination 
+
+>[!NOTE]
+>
+>The [!DNL Acxiom Audience Distribution] destination is in beta. This destination connector and documentation page are created and maintained by the [!DNL Acxiom] team. For any inquiries or update requests, contact Acxiom directly [here](mailto:acxiom-adobe-help@acxiom.com).
+
+Use the [!DNL Acxiom Audience Distribution] destination to enhance audiences with [!DNL Acxiom's Real ID] technology and activate audiences to multiple platforms, such as [!DNL Altice], [!DNL Ampersand], [!DNL Comcast], and more.

```

Comment:
Is there link you can add to Real ID where readers can learn more about what this ID is and what it does?

Reply:
Acxiom is currently working on this content and we will update the documentation to include a link as soon as it is complete.

Reply:
Done. Thank you!